hack/update-codegen.sh

```diff
@@ -49,6 +49,7 @@ function gen() {
     --input-dirs ./pkg/apis/v1alpha1/ \
     --input-dirs ./pkg/apis/config/v1alpha1/ \
     --input-dirs ./pkg/apis/action/v1alpha1/ \
+    --output-package sigs.k8s.io/kwok/pkg/apis \
```

Comment:
Is your environment unable to execute this script?

Reply:
Yes, I can't update, but this change might not be necessary because KwokctlConfiguration is not changed. So I restored.
Let's discuss this problem somewhere else.